src/main/resources/roleconfig/role_employment.json

```diff
@@ -0,0 +1,92 @@
+[
+  {
+    "name": "specific-access-approver-legal-operations",
```

Comment:
this is not require.  This is same as specific-access-approver-legal-ops in role_common.json. In case any name correction then it has to be corrected in confluence